src/admin/project-insights.js

```diff
@@ -0,0 +1,63 @@
+window.c.admin.ProjectInsights = (function(m, c, models){
```

Comment:
I think it's better to use a new namespace, as this is not part of the admin, but is more related to project management. Maybe we should have a namespace called projects. @vicnicius any thoughts?